lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1086,6 +1086,32 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'Controls auto-instrumentation of the Tilt template rendering library at start up. May be one of [auto|prepend|chain|disabled].'
         },
+        :'instrumentation.grpc_client' => {
+          :default => instrumentation_value_of(:disable_grpc_client),
+          :documentation_default => 'auto',
+          :public => true,
+          :type => String,
+          :dynamic_name => true,
+          :allowed_from_server => false,
+          :description => 'Controls auto-instrumentation of gRPC clients at start up.  May be one of [auto|prepend|chain|disabled].'
+        },
+        :'instrumentation.grpc.host_denylist' => {
+          :default => [],
+          :public => true,
+          :type => Array,
+          :allowed_from_server => false,
+          :transform => DefaultSource.method(:convert_to_regexp_list),
+          :description => 'Define grpc connections you want the agent to ignore by specifying a list of patterns matching the hostname(s) of grpc servers to be ignored'
```

Comment:
This description should match the description in newrelic.yml

Reply:
addressed with 3f5f9af98a932478ebcc264689da9bb278986607